### ISSUE TITLE
Accessibility:List: The “unread” information is not conveyed when unread list items are accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3487]
Description: The “unread” information is not conveyed when unread list items are accessed. 

Expected: Unread information about list items should be conveyed to the user, when list items are accessed. 

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94649362-23367c00-0312-11eb-9235-0dacd0134e18.png)
